### PR TITLE
Add VueMatomo analytics

### PR DESCRIPTION
**Pull request by daine**

### CODE REVIEW

File: app/frontend/src/main.js [hunk 1]
```diff
@@ -72,6 +73,16 @@ Vue.use(VueAnalytics, {
   })
 })
 
+
+if (isProduction()) {
+  Vue.use(VueMatomo, {
+    host: 'https://water-matomo.apps.silver.devops.gov.bc.ca/',
+    siteId: 2,
+    router: router,
+    domains: '*.silver.devops.gov.bc.ca'
```
Comment (stephenhillier) [inline app/frontend/src/main.js:82]: https://apps.nrs.gov.bc.ca

File: app/frontend/src/main.js [hunk 2]
```diff
@@ -19,6 +19,7 @@ import Vuex, { mapActions } from 'vuex'
 import VueNoty from 'vuejs-noty'
 import BootstrapVue from 'bootstrap-vue'
 import VueAnalytics from 'vue-analytics'
+import VueMatomo from 'vue-matomo'
```
Comment (stephenhillier) [inline app/frontend/src/main.js:22]: Add to package.json?

**Review by stephenhillier — APPROVED** [review]
Overall looks good with a couple minor changes (update the domains in the VueMatomo options to use our prod domain, and save the vue-matomo package to package.json)

**Comment by daine** [comment]
> Overall looks good with a couple minor changes (update the domains in the VueMatomo options to use our prod domain, and save the vue-matomo package to package.json)

Thanks for catching those! New commit resolves those two issues.

**Pull request merged**